src/popup/modules/UserInterface.js

```diff
@@ -591,72 +629,109 @@ export function init() {
-        if (qrCodeSize.sizeType === "remember" && qrCodeSize.hasOwnProperty("sizeText")) {
-            console.info("restore qr code text size:", qrCodeSize.sizeText);
-            // is saved as CSS string already
-            // height is NOT (anymore) restored, as this may cause display errors (likely due to different content-box settings) and the height does not matter, anyway
-            qrCodeText.style.width = qrCodeSize.sizeText.width;
-
-            // detect too small size
-            const minimalSize = qrCodeSize.size + parseInt(qrCodeSize.sizeText.height, 10);
-            if (window.innerHeight < minimalSize) {
-                console.error("too small size", window.innerHeight, "should be at least: ", minimalSize);
+            // also set height of text (also to prevent display errors) when remember is enabled
+            if (
+                qrCodeSize.sizeType === "remember" &&
+                qrCodeSize.hasOwnProperty("sizeText")
+            ) {
+                console.info("restore qr code text size:", qrCodeSize.sizeText);
+                // is saved as CSS string already
+                // height is NOT (anymore) restored, as this may cause display errors (likely due to different content-box settings) and the height does not matter, anyway
+                qrCodeText.style.width = qrCodeSize.sizeText.width;
+
+                // detect too small size
+                const minimalSize =
+                    qrCodeSize.size + parseInt(qrCodeSize.sizeText.height, 10);
+                if (window.innerHeight < minimalSize) {
+                    console.error(
+                        "too small size",
+                        window.innerHeight,
+                        "should be at least: ",
+                        minimalSize
+                    );
+                }
             }
         }
-    });
+    );
 
     const initQrTypespecificSettings = createContextMenu();
 
     // return Promise chain
-    return Promise.all([applyingMonospaceFont, applyingQrSize, applyingQrColor, initQrTypespecificSettings]);
+    return Promise.all([
+        applyingMonospaceFont,
+        applyingQrSize,
+        applyingQrColor,
+        initQrTypespecificSettings,
+    ]);
 }
 
-function KeyPress(e) {
-    var evtobj = window.event? event : e
-    if (evtobj.keyCode == 61 && evtobj.ctrlKey) {
-        setNewQrCodeSize(qrLastSize+20, true);
-    };
-    if (evtobj.keyCode == 173 && evtobj.ctrlKey) {
-        setNewQrCodeSize(qrLastSize-20, true);
-    };
-}
+// zoom listener
+document.onkeydown = zoomQrCode;
 
-document.onkeydown = KeyPress;
+/**
+ * Listens to Alt + plus/minus keypresses to alter 
+ * QR code size.
+ *
+ * @function
+ * @returns {void}
+ */
+function zoomQrCode(e) {
+    var evtobj = window.event ? event : e;
+    if (evtobj.keyCode == 61 && evtobj.altKey && qrLastSize < 440) {
+        setNewQrCodeSize(qrLastSize + 30, true);
```

Comment:
Note `setNewQrCodeSize` may – depending on https://github.com/rugk/offline-qr-code/blob/220d6f3c7394ba4dc5a8d831edd3807838041c9a/src/popup/modules/UserInterface.js#L242 save the size to add-on storage's, too. THough thinking about it, that makes sense, because it is then just persisted.

So yeah, I dunno whether my idea of remembering it additionally, was a good idea, but it seems okay'ish.